packages/sit-onyx/src/components/OnyxTag/OnyxTag.ct.tsx

```diff
@@ -0,0 +1,78 @@
+import { expect, test } from "../../playwright-axe";
+import { ONYX_COLORS } from "../../types/colors";
+import { executeScreenshotsForAllStates, mockPlaywrightIcon } from "../../utils/playwright";
+import OnyxTag from "./OnyxTag.vue";
+
+/// [TODO] Skipped Reason: Need to get the color information to pass this analyze
+test.skip("should render", async ({ mount, makeAxeBuilder }) => {
```

Comment:
Alternatively you can also move the accessibility test to your tests below and just comment it out :)